dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -124,6 +126,23 @@ public DdlStatementExecution createDdlStatementExecution(String ddl)
         return new DdlStatementExecution(getConnection().createStatement(), ddl);
     }
 
+    @Override
+    public StatementExecution createStatementExecution(PreparedStatement statement, boolean clearParameters) {
+        return new StatementExecution(statement, clearParameters);
+    }
+
+    @Override
+    public StatementExecution createFunctionStatementExecution(PreparedStatement statement, boolean clearParameters) {
+        return new StatementExecution(statement, clearParameters);
+    }
+
+    protected TypeTransformerFactory toJdbcCompatibleValue = new TypeTransformerFactory();
```

Comment:
`toJdbcCompatibleValue` is OK for the accessor method (what we discussed) but here I'm not sure. At first glimpse I think previous name here was better.